lib/paths-cache.js

```diff
@@ -223,13 +224,30 @@ export default class PathsCache extends EventEmitter {
     this._cancelled = false
     this.emit('rebuild-cache')
 
-    return execPromise('find --version')
-      .then(
+    // check if find is available and store its path
+    if (this.hasFind == null) {
```

Comment:
Is the goal of looking for `find` first to speed it up? I feel like `process.platform === 'win32'` will be a lot faster than using which. Also it seems like letting `this._buildInitialCacheWithFind()` fail would be faster than this method as well.

Reply:
Also windows does have a `find` just not the one we are looking for.

Reply:
Well, find is available on Windows. Only `find --version` fails on it. We can still use _buildInitialCacheWithFind on Windows, I believe.

Reply:
`_buildInitialCacheWithFind` will always fail on windows since `find` on windows does not take the same parameters as `find` on linux.

[find docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/find)

Reply:
Where do we call find? We should use correct parameters on Windows. I could not find it.

Reply:
we call `find` in the function `_buildInitialCacheWithFind`.

[linux find docs](https://ss64.com/bash/find.html)

[windows find docs](https://ss64.com/nt/find.html)

Reply:
We can fix the parameters later. Currently, #243 is passing the tests. Should we merge that for now?

Reply:
sounds good to me.